src/app.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * This program has been developed by students from the bachelor
+ * Computer Science at Utrecht University within the Software Project course.
+ *
+ * © Copyright Utrecht University
+ * (Department of Information and Computing Sciences)
+ */
+
+import express, { NextFunction } from "express";
+import { Request, Response } from "express";
+import { Context } from "./context";
+import { installRoute, uninstallRoute } from "./install";
+import { body } from "express-validator";
+import { asyncCatch } from "./utils";
+import { handleValidationResult } from "./validate";
+
+export function buildApp(ctx: Context) {
+  const app = express();
+
+  app.use(express.json());
+
+  app.post(
```

Comment:
It might be nice to put "install" and "uninstall" in a router that is exported alongside the functions in install.ts